### PR TITLE
Add section on wiping the data disk

### LINKED ISSUE #30654: How do I reset a device running on Raspi after losing the username/password?
### Feedback

The online documentation tells me 

"If you do not have a backup, resetting the device will erase all data."

But I don't find info about how to do that.

I can access a "ha" cli, but I can't manage to stop a "running dmesg log", therefore I can' log in as root.
But even if I could, what are the commands to reset the device?

I am a complete newbie to HASS, please help me with this! G**gle didn't give me a clear route...

### URL

https://www.home-assistant.io/docs/locked_out/

### Version

2024.1.0

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Add section on wiping the data disk

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/supervisor/pull/4934
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #30654

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/common-tasks/commandline.md [hunk 1]
```diff
@@ -100,7 +100,19 @@ Use "ha [command] --help" for more information about a command.
...
 
 You can also access the Home Assistant Operating System via a directly connected keyboard and monitor, the console.
+
+#### Wiping the data disk wipe from the command line
+
+In Home Assistant Operating System, there is a dedicated command to delete all user data as well as Home Assistant Core, Supervisor, and any installed add-ons: this command is `ha os datadisk wipe`.
+
+The command `ha os datadisk wipe` marks the data partition (either internal on the eMMC or the SD card, or on an external attached data disk) as to be cleared on the next reboot. The command automatically reboots the system. Upon reboot, the data is cleared. Then the system continues to boot and reinstalls the latest version of all Home Assistant components.
+
+The `ha os datadisk wipe` command can be run from the local terminal. Connect a display and keyboard and use the terminal.
```
Comment (agners) [inline source/_includes/common-tasks/commandline.md]:
```suggestion
The `ha os datadisk wipe` command can only be run from the local terminal. Connect a display and keyboard and use the terminal.
```

File: source/_includes/common-tasks/commandline.md [hunk 2]
```diff
@@ -100,7 +100,19 @@ Use "ha [command] --help" for more information about a command.
...
+
+Note, if you have a Home Assistant Yellow, there is a the red button to wiipe the data disk. Follow the procedure on [resetting the device](https://yellow.home-assistant.io/guides/factory-reset/).
```
Comment (agners) [inline source/_includes/common-tasks/commandline.md]:
```suggestion
Note, if you have a Home Assistant Yellow, there is a the red button to wipe the data disk. Follow the procedure on [resetting the device](https://yellow.home-assistant.io/guides/factory-reset/).
```

File: source/_includes/common-tasks/commandline.md [hunk 3]
```diff
@@ -100,7 +100,19 @@ Use "ha [command] --help" for more information about a command.
...
 
 {% if page.installation == "os" %}
+
 ### Console access
 
 You can also access the Home Assistant Operating System via a directly connected keyboard and monitor, the console.
+
+#### Wiping the data disk wipe from the command line
+
+In Home Assistant Operating System, there is a dedicated command to delete all user data as well as Home Assistant Core, Supervisor, and any installed add-ons: this command is `ha os datadisk wipe`.
```
Comment (agners) [inline source/_includes/common-tasks/commandline.md]: Reads a bit weird to me. Maybe?
```suggestion
In Home Assistant Operating System, the dedicated command `ha os datadisk wipe` deletes all user data as well as Home Assistant Core, Supervisor, and any installed add-ons.
```

Also doesn't make me entirely happy though, as the sentence gets a bit long.

**Review by agners — CHANGES_REQUESTED** [review]

File: source/_includes/common-tasks/commandline.md [hunk 4]
```diff
@@ -100,7 +100,19 @@ Use "ha [command] --help" for more information about a command.
...
 
 {% if page.installation == "os" %}
+
 ### Console access
 
 You can also access the Home Assistant Operating System via a directly connected keyboard and monitor, the console.
+
+#### Wiping the data disk wipe from the command line
+
+In Home Assistant Operating System, the dedicated command `ha os datadisk wipe` deletes all user data as well as Home Assistant Core, Supervisor, and any installed add-ons.
```
Comment (c0ffeeca7) [inline source/_includes/common-tasks/commandline.md]:
```suggestion
In Home Assistant Operating System, there is a dedicated command to wipe the data disk. `ha os datadisk wipe` deletes all user data as well as Home Assistant Core, Supervisor, and any installed add-ons.
```

File: source/_includes/common-tasks/commandline.md [hunk 5]
```diff
@@ -100,7 +100,19 @@ Use "ha [command] --help" for more information about a command.
...
 
 {% if page.installation == "os" %}
+
 ### Console access
 
 You can also access the Home Assistant Operating System via a directly connected keyboard and monitor, the console.
+
+#### Wiping the data disk wipe from the command line
+
+In Home Assistant Operating System, there is a dedicated command to wipe the data disk. `ha os datadisk wipe` deletes all user data as well as Home Assistant Core, Supervisor, and any installed add-ons.
```
Comment (agners) [inline source/_includes/common-tasks/commandline.md]:
```suggestion
In Home Assistant Operating System, the `ha os datadisk wipe` command wipes the data disk. The command deletes all user data as well as Home Assistant Core, Supervisor, and any installed add-ons.
```

File: source/_includes/common-tasks/commandline.md [hunk 6]
```diff
@@ -100,7 +100,19 @@ Use "ha [command] --help" for more information about a command.
...
+
+#### Wiping the data disk wipe from the command line
+
+In Home Assistant Operating System, the `ha os datadisk wipe` command wipes the data disk. The command deletes all user data as well as Home Assistant Core, Supervisor, and any installed add-ons.
+
+The command `ha os datadisk wipe` marks the data partition (either internal on the eMMC or the SD card, or on an external attached data disk) as to be cleared on the next reboot. The command automatically reboots the system. Upon reboot, the data is cleared. Then the system continues to boot and reinstalls the latest version of all Home Assistant components.
+
+The `ha os datadisk wipe` command can only be run from the local terminal. Connect a display and keyboard and use the terminal.
+
+Note, if you have a Home Assistant Yellow, there is a the red button to wipe the data disk. Follow the procedure on [resetting the device](https://yellow.home-assistant.io/guides/factory-reset/).
```
Comment (c0ffeeca7) [inline source/_includes/common-tasks/commandline.md]:
```suggestion
Note, if you have a Home Assistant Yellow, there is a red hardware button to wipe the data disk. Follow the procedure on [resetting the device](https://yellow.home-assistant.io/guides/factory-reset/).
```

**Review by agners — APPROVED** [review]

**Pull request merged**